src/main/java/seedu/address/logic/commands/UpdateSecLevelCommand.java

```diff
@@ -49,26 +55,21 @@ public UpdateSecLevelCommand() {
     public CommandResult execute(Model model) throws CommandException {
         requireNonNull(model);
         if (isUndo) {
-            if (beforeLastUpdateStudents == null) {
+            if (beforeLastUpdateStudents.size() == 0) {
```

Comment:
Nice changes to solve the issue where undolevel can still be called even when there is no student data in Tutorium.